distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -1438,7 +1438,7 @@ type Column
              column.text_replace '"(.*?)"'.to_regex '($1)'
     @term make_regex_text_widget
     text_replace : Text | Regex | Column -> Text | Column -> Case_Sensitivity -> Boolean -> Column
-    text_replace self term="" new_text="" case_sensitivity=Case_Sensitivity.Sensitive only_first=False =
+    text_replace self (term : Text | Regex | Column = "") new_text="" case_sensitivity=Case_Sensitivity.Sensitive only_first=False =
```

Comment:
Shouldn't there be similar changes to `DB_Column`?

Reply:
I believe DB_Column had it right, but I will double check